src/styles/_EventContainer.scss

```diff
@@ -1,5 +1,9 @@
 .EventContainer {
-  border: 1px solid black;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(400px, 400px));
```

Comment:
I haven't used the auto-fit before - it looks great and I'm excited that it's so easily mobile responsive!